examples/inaccessible.py

```diff
@@ -0,0 +1,32 @@
+import graphene
+
+from graphene_federation import inaccessible, external, provides, key, override
+
+from graphene_federation import build_schema
+
+
+@key(fields="x")
+class Position(graphene.ObjectType):
+    x = graphene.Int(required=True)
+    y = external(graphene.Int(required=True))
+    z = inaccessible(graphene.Int(required=True))
+    a = provides(graphene.Int(required=True), fields="x")
+    b = override(graphene.Int(required=True), _from="h")
```

Comment:
```suggestion
    b = override(graphene.Int(required=True), from_="h")
```

@erikwrede do we have a preference on this? I usually do `from_` and that's what I've seen usually done :) 

`_something` is more for private thing, no?

Reply:
You are correct @patrick91 , will do the changes.

Reply:
Agreed!